assets/wizards/readerRevenue/views/donation/index.tsx

```diff
@@ -159,13 +192,35 @@ export const DonationAmounts = () => {
 								currencySymbol={ currencySymbol }
 								label={ section.staticLabel }
 								value={ amounts[ section.key ][ 3 ] }
+								min={
+									amounts[ section.key ][ 3 ] >= minimumDonationFloat
+										? amounts[ section.key ][ 3 ]
+										: minimumDonationFloat
+								}
```

Comment:
`min` attribute should have the value of `minimumDonationFloat`, shouldn't it? And then the `error` attribute should compare the input value and the minimum. Currently the error message won't be displayed if "Untiered" selection is made.

Reply:
Whoops, that was an oversight. 15143df should fix both of these issues.